### DISCUSSION TITLE
Pistol Round configurations

### DISCUSSION THREAD

**Discussion by billygoat34** [yonilerner/cs2-retakes-allocator#113] [Ideas]
Currently you get a pistol, kevlar, and a nade which can total to over $1650 when players should be limited to $800 on pistol rounds.
It'd be better imo if you could choose to get `default pistol + vest`, or `upgraded pistol + one nade`, or `default pistol + kit + nade` (for CTs)

**Comment by yonilerner** [comment]
Still working on this, sorry for the delay!

**Reply by yonilerner** [reply]
Im not sure I will get around to this any time soon, but I left some notes here about how I would do this:

https://github.com/yonilerner/cs2-retakes-allocator/issues/66#issuecomment-2558351574
